### ISSUE TITLE
Plotly support for vector tiles

### ISSUE THREAD

**Issue by morgana2313** [ghtmtt/DataPlotly#307]
**Describe the solution you'd like**
Plotly doesn't seem to support graphing data from [Vector Tile layers](https://docs.qgis.org/3.22/en/docs/user_manual/working_with_vector_tiles/vector_tiles_properties.html). The Layer selector only shows other layers (like shapefile or WFS)

**Describe alternatives you've considered**
I can't think of any other than not using vector tiles.

**Additional context**

* example vector tile layer [US ZIP](https://geoplatform.github.io/geoplatform-demos/qgis-example.html)
* example WFS layer [dutch windmills](https://demo.pygeoapi.io/stable/collections/dutch_windmills?f=json)
* example shape layer [ne_10m_admin_0_boundary_lines_map_units](https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_boundary_lines_map_units.zip)

![Screenshot 2022-11-29 at 14 13 33](https://user-images.githubusercontent.com/32460045/204538823-990f7f8b-886e-4a09-9d78-0cd874d5f844.png)

**Comment by ghtmtt** [comment]
correct, it is not supported yet. Thanks for the idea, this would be a great enhancement
